Sources/Plasma/PubUtilLib/plAvatar/plAvBrainHuman.cpp

```diff
@@ -472,14 +474,23 @@ bool plAvBrainHuman::MsgReceive(plMessage * msg)
 bool plAvBrainHuman::IHandleClimbMsg(plClimbMsg *msg)
 {
     bool isStartClimb = msg->fCommand == plClimbMsg::kStartClimbing;
+
     if(isStartClimb)
     {
-        // let's build a seek task to get us to the attach point
-        plKey seekTarget = msg->fTarget;
-        plAvTaskSeek *seekTask = new plAvTaskSeek(seekTarget);
-        QueueTask(seekTask);
+        // Warp the player to the Seekpoint
+        plSceneObject *avatarObj = plSceneObject::ConvertNoRef(plNetClientMgr::GetInstance()->GetLocalPlayer());
+        plSceneObject *obj = plSceneObject::ConvertNoRef(msg->fTarget->ObjectIsLoaded());
+        plArmatureMod *localAvatar = plAvatarMgr::GetInstance()->GetLocalAvatar();
+        plArmatureMod *climbAvatar = plArmatureMod::ConvertNoRef(fArmature);
+        if (climbAvatar == localAvatar) // is it our avatar who has to seek?
+        {
+            hsMatrix44 target = obj->GetLocalToWorld();
+            plWarpMsg *warp = new plWarpMsg(nullptr, avatarObj->GetKey(), plWarpMsg::kFlushTransform, target);
+            warp->SetBCastFlag(plMessage::kNetPropagate);
+            plgDispatch::MsgSend(warp);
+        }
```

Comment:
Did the seeking not work?

Reply:
current seeking can randomly crash, or place the avatar further down than the start point when starting to climb on the wall.